doc/content/integrations/pubsub/message-types/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Message Types"
+description: ""
+---
+
+The Application Server can be configured to **publish** messages for any of the following events:
+- Downlink ack
+- Downlink failed
+- Downlink nack
+- Downlink queued
+- Downlink sent
+- Join accept
+- Location solved
+- Service data
+- Uplink message
```

Comment:
Okay yeah sorry about that comment about keeping this sorted. I didn't mean alphabetic but logical, so basically how it was.